secrets/secrets.go

```diff
@@ -39,7 +39,7 @@ func GetPort() string {
 	// Find port from env var or default to 80
 	port, exists := os.LookupEnv("PORT")
 	if !exists {
-		port = "80"
+		port = "8080"
```

Comment:
wait why, if u need to change it just set the port env var?

Reply:
Oh that was for testing purposes, will change back